public/js/jquery.t.js

```diff
@@ -16,6 +16,17 @@
         });
     };
 
+    var setupResponsiveness = function ($queryDiv, $graphDiv, index, opts)  {
+        var debounced_draw = _.debounce(function () {
+            var shownHits = $queryDiv.find('.ghit > g').length;
+            $.graphIt($queryDiv, $graphDiv, shownHits, 0, opts);
+        }, 125);
+        // Bind resize event only when the SVG is created for the first time.
+        if (index === 0) {
```

Comment:
I wouldn't have raised this if I hadn't found the above bug:

I see `if (index === 0)` check at L356 as well (when your patch is applied). Is it not enough to call `setupResponsiveness` in the same conditional?